src/sections/services.html

```diff
@@ -1,11 +1,14 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>services</title>
-</head>
-<body class="font-outfit">
-    <h1 class="text-blue-500">Services</h1>
-</body>
-</html>
\ No newline at end of file
+<div class="grid my-5 grid-cols-1 gap-20 px-8 md:px-20 font-outfit lg:grid-cols-2">
+    <div class="flex flex-col gap-y-5 text-2xl md:text-4xl lg:gap-y-14">
+        <h5 class="font-semibold ">Brand strategy</h5>
+        <h5 class="font-semibold ">Corporate Identity & <br class="md:hidden"> Website development</h5>
+        <h5 class="font-semibold ">SEO & copywriting</h5>
+        <h5 class="font-semibold ">Media Relations</h5>
+        <h5 class="font-semibold ">Digital marketing &<br class="md:hidden">content production</h5>
+        <h5 class="font-semibold ">Influencer relations</h5>
```

Comment:
- [ ]  Are they the titles or the list? How do you see it as a group?

Reply:
On my side, it would be better if they were <li>s in a <ul>.

Reply:
Yes, that is what I see, too. you can check in with a group to see if you can implement it.